CONTRIBUTING.rst

```diff
@@ -126,8 +126,7 @@ These are the basic steps needed to start developing on Sphinx.
 
    * Build the documentation and check the output for different builders::
 
-       cd doc
-       make clean html latexpdf
+       make docs target="clean html latexpdf"
```

Comment:
For reference, I'd like to rewrite this entire document to use tox instead, meaning this would become:

    tox -e docs -- -b clean html latexpdf

That doesn't mean this shouldn't go in as-is now though

Reply:
would the resulting files lie in the virtual environment?

Reply:
yeah, they do.